.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Generate release-artifacts
+
+on:
+  push:
```

Comment:
It might be worth including `workflow_dispatch:` in this list as well so the workflow can be triggered manually.